filament/backend/src/vulkan/VulkanHandles.h

```diff
@@ -143,7 +143,9 @@ struct VulkanTexture : public HwTexture {
     void setPrimaryRange(uint32_t minMiplevel, uint32_t maxMiplevel);
 
     // Gets or creates a cached VkImageView for a range of miplevels and array layers.
-    VkImageView getImageView(VkImageSubresourceRange range);
+    // If force2D is true, this always returns VK_IMAGE_VIEW_TYPE_2D, regardless of the
```

Comment:
it's not so much that it returns that constant right? It returns an VkImageView of that type, correct?

Reply:
Correct, I tweaked the wording.